nextcord/types/protocols/abc.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
We dont use abcs?

Reply:
> We dont use abcs?

I asked and was told to make one for Messageable and other stuff like that?